### DISCUSSION TITLE
Error with importing Shapefile and string to float

### DISCUSSION THREAD

**Discussion by coolbeansnoah** [cnr-isti-vclab/TagLab#126] [Q&A]
When I try to import any shape file from Agisoft it rejects it every time. I have even tried exporting it out of TagLab into Agisoft and back to TagLab to see if it's something in the way it's formatted but it fails every time. If there are any tips or advice you can give that would be helpful.
![Screenshot 2023-12-19 094156](https://github.com/cnr-isti-vclab/TagLab/assets/73969452/6e2149d0-f485-43c2-9f57-5ec9dbc407bb)
I know that the string to float is caused when I entered the measurements of pixel size with a comma instead of a period. it still let me edit the project and everything at first, but now when I want to gather the data that uses comparisons from other time points the program always crashes and I believe it's rooted to the string to float. Is there a way to remove a single map from the project that you are working on?

**Comment by dpoursanidis** [comment]
Have you open it in QGIS and make new field as string and copy the data from the desired column? That might be a workaround solution.

**Reply by coolbeansnoah** [reply]
No, I have not. I will give that a try and see how it goes. 
Thanks!

**Reply by coolbeansnoah** [reply]
The Qgis method didn't work, but I was able to figure out the "Float" problem by going into the .json file with notepad and searching "map_px_to_mm_factor" and editing the "," to "." and save the notepad. Just open it how you would any other project and it worked no problem.
